ci/aws/aws-cluster.lokocfg.envsubst

```diff
@@ -136,9 +136,11 @@ component "prometheus-operator" {
   }
 }
 
-component "experimental-linkerd" {
-  enable_monitoring   = true
-}
+# Skip testing Linkerd until they upgrade to go1.15. This is to avoid the errors like:
```

Comment:
We could make `GODEBUG=x509ignoreCN=0` an optional flag, as `linkerd` may not be the only one affected by this issue. User workloads may also hit this.

Reply:
This should be added in our build code? Like as an env var before we do `go build` or this is something Linkerd has to do?

Reply:
I guess that would be an optional env variable for self-hosted kube-apiserver? But I'm not 100% sure we should add it, just something to consider and decide.

Reply:
added an env var with a flag.